src/popup/router/components/Modals/Default.vue

```diff
@@ -4,13 +4,27 @@
     close
     @close="resolve"
   >
-    <template
-      v-if="title"
-      slot="header"
-    >
-      {{ title }}
+    <template slot="header">
+      <Component
+        :is="icon"
+        v-if="icon"
+        class="icon"
+        :class="type"
+      />
+      {{ title || $t(`modals.${type}`).title || '' }}
```

Comment:
I think that there shoul be `no-dynamic-keys`, since this PR is called `Rebrand modals` we may fix it here

Reply:
That approach wasn't introduced in this PR and is used at a few places. Any idea why `vue-i18n/no-dynamic-keys` does not detect an error here?

To avoid using dynamic keys, I must create separate components for each modal type. Please advise.

Reply:
Main reason was that some modal messages need new lines and it requires component interpolation. I separated the only use case so far in a component: https://github.com/aeternity/superhero-wallet/pull/1100/files#diff-bf1f126eebbafd355e30fb983bcc776954252488c787cb5448ee0b543a87f296R7-R13

The rest can use the default as others.

Reply:
There are only prior usages of modal with `type` now and I'm going to do a refactor in this PR or another.

Reply:
#1150